lib/typescript/httpclient/yarn.lock

```diff
@@ -0,0 +1,32 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I am wondering if it makes sense to commit this file since we don't use it for the bazel builds

Reply:
makes sense! will take it out!